src/sentry/static/sentry/app/views/organizationIntegrations/index.jsx

```diff
@@ -56,7 +56,10 @@ export default class OrganizationIntegrations extends AsyncComponent {
         i => i.provider.key == provider.key
       );
       const isInstalled = integrations.length > 0;
-      return Object.assign({integrations, isInstalled}, provider);
+      let newProvider = Object.assign({}, provider);
+      newProvider.integrations = integrations;
+      newProvider.isInstalled = isInstalled;
+      return newProvider;
```

Comment:
Wait why?

Reply:
:man_shrugging: to be honest. Tests were failing the other way – the `integrations` key wasn't being set.

Reply:
was the `isInstalled` key being set? Was `integrations` undefined?

Reply:
`integrations` was an array and definitely had items in it – which is why changing to this dumb syntax fixed things.

Seems like the `Object.assign` + object-shorthand syntax wasn't doing what you'd expect during tests, for whatever reason.

Reply:
the other way means that `provider.integrations` will overwrite the local `integrations`?

I don't know which way is correct, but object spread might be more clear and concise:

```
return {
...provider,
integrations,
isInstalled,
};
```

Reply:
Good point, lets just use object spread.

Reply:
Ugh, yea, I had the stupid arguments backwards. I feel it should have still worked. Whatever. Thanks for the JS-fu.